### PR TITLE
Add flake8 to linters

### PR DESCRIPTION
**Pull request by cassios**
Replace isort with flake8-isort.

Run pre-commit on all files.

### CODE REVIEW

**Comment by cassios** [comment]
Adding flake8 to get more verifications

File: pyproject.toml [hunk 1]
```diff
@@ -16,3 +16,9 @@ skip-string-normalization = true # don't force double quotes
 [tool.isort]
 profile = "hug"
 line_length = 99
+
+[tool.flake8]
```
Comment (pdonorio) [inline pyproject.toml:20]: is this supported?

https://github.com/PyCQA/flake8/issues/234

Comment (pdonorio) [reply pyproject.toml:20]: never mind, you used `pyproject-flake8`

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -8,8 +8,9 @@ repos:
...
-- repo: https://github.com/timothycrosley/isort
-  rev: 5.10.1
+- repo: https://gitlab.com/pycqa/flake8
+  rev: 4.0.1
   hooks:
-  - id: isort
-    args: [--sp=pyproject.toml]
+  - id: flake8
+    entry: pflake8
+    additional_dependencies: [pyproject-flake8, flake8-isort]
```
Comment (pdonorio) [inline .pre-commit-config.yaml:16]: doesn't flake8-isort only checks and not fix the imports?

**Review by pdonorio — APPROVED** [review]
Thanks! <3

**Pull request merged**